examples/post-sync/.golangci.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+run:
+  timeout: 5m
+
+linters:
+  enable:
+  - gofmt
+  - goimports
+  - revive
+  - staticcheck
+  - unconvert
+  - misspell
+
+issues:
+  exclude-rules:
+    - linters:
+        - revive
+      text: "should have a package comment"
+    - linters:
+        - revive
+      text: "error strings should not be capitalized" 
```

Comment:
nit: trailing linebreak (maybe add to the one you copied from too, if it's missing)

Reply:
done